me.stonegate.tsacdop.yml

```diff
@@ -0,0 +1,36 @@
+app-id: "me.stonegate.tsacdop"
+runtime: org.gnome.Platform
+runtime-version: "40"
```

Comment:
does this work on 41? that's newer so has more time in security support

Reply:
also since this is flutter based, I wonder if freedesktop sdk isn't enough. (unless it needs GtkWebKit or other GNOME specific stuff)

Reply:
I will update it to 41, should work well.

Reply:
Build failed under runtime 41, I have to keep on 40 right now.